src/driver/libxdma.c

```diff
@@ -2414,25 +2396,10 @@ static int probe_engines(struct xdma_dev *xdev)
 	return 0;
 }
 
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(3,5,0)
 static void pci_enable_capability(struct pci_dev *pdev, int cap)
 {
 	pcie_capability_set_word(pdev, PCI_EXP_DEVCTL, cap);
 }
-#else
```

Comment:
You can probably cleanup and open code pci_enable_capability() now....

This code has way too many function I'd expect to be in the PCI core based on their name; but are actually private functions.

Reply:
Yup, good idea. Thanks